test/common/wpt.js

```diff
@@ -402,6 +404,29 @@ const kIncomplete = 'incomplete';
 const kUncaught = 'uncaught';
 const NODE_UNCAUGHT = 100;
 
+const limit = (concurrency) => {
+  let running = 0;
+  const queue = [];
+
+  const execute = async (fn) => {
+    if (running < concurrency) {
+      running++;
+      try {
+        await fn();
+      } finally {
+        running--;
+        if (queue.length > 0) {
+          execute(queue.shift());
+        }
```

Comment:
This is not returned or awaited. Is there a risk of unhandled rejection?

Reply:
It behaves as expected even with the WPT suite updated (like the daily WPT job does).

This is the await in the queue.
```js
await events.once(worker, 'exit').catch(() => {});
```

That being said, I am aware it's just a rudimentary concurrency limit function...